pkg/reconciler/kubernetes/tektoninstallerset/install.go

```diff
@@ -151,6 +152,7 @@ func (i *installer) EnsureNamespaceScopedResources() error {
 			triggerTemplatePred,
 			servicePred,
 			routePred,
+			statefulSetPred,
```

Comment:
@nikhil-thomas shall we maintain a lifecycle for StatefulSet as well, just like we do for Deployments?

Reply:
we can definitely check if they are up and running as a strater. However, im not keen on adding  a new installerStatus for each new item.

May be we should combine all validations under a single status field.

Reply:
yeah. In a follow-up PR will address that